src/m365/spo/commands/listitem/listitem-get.ts

```diff
@@ -49,16 +49,16 @@ class SpoListItemGetCommand extends SpoCommand {
       `${args.options.webUrl}/_api/web/lists(guid'${formatting.encodeQueryParameter(listIdArgument)}')`
       : `${args.options.webUrl}/_api/web/lists/getByTitle('${formatting.encodeQueryParameter(listTitleArgument)}')`);
 
-    const propertiesSelect: string = args.options.properties ?
-      `?$select=${encodeURIComponent(args.options.properties)}` :
-      (
-        (!args.options.output || args.options.output === 'text') ?
-          `?$select=Id,Title` :
-          ``
-      );
+    const propertiesSelect: string[] = args.options.properties ? args.options.properties.split(",")
+      : (!args.options.output || args.options.output === "text") ? ["Title", "Id"] : [];
```

Comment:
Default output is JSON so will this work correctly, because no output and output === 'text' are different?

Reply:
Hi @waldekmastykarz 
Tested it with all output, seems to be working fine.

![image](https://user-images.githubusercontent.com/8310319/176461253-b27cf782-8879-46ef-bc3a-db176c000165.png)